test/system/045-start.bats

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bats   -*- bats -*-
+
+load helpers
+
+function setup() {
+    basic_setup
+    run_podman run -d $IMAGE /bin/true
+    cid_none="$output"
+    run_podman run -d --restart=always $IMAGE /bin/true
+    cid_always="$output"
+}
+
+function teardown() {
+    basic_teardown
+}
+
+@test "podman start --all - start all containers" {
+    run_podman start --all
+    is "$output" "$cid_none
+$cid_always"
+}
+
+@test "podman start --all --latest - should fail" {
+    run_podman 125 start --all --latest
```

Comment:
This is probably going to fail with podman-remote (I could check logs, but I'm lazy). And, since the error messages are the same, you could probably combine these two tests into one:
```bash
@test "podman start --all with incompatible options" {
    expect="Error: either blah blah"

    run_podman 125 start --all 12345
    is "$output" "$expect" "podman start --all + CID"

    if ! is_remote; then
        run_podman 125 start --all --latest
        is "$output" "$expect" "podman start --all --latest"
    fi
```